.github/workflows/nightly-mjx-build.yaml

```diff
@@ -0,0 +1,143 @@
+name: Nightly MJX build
+run-name: Nightly MJX build (${{ github.event_name == 'workflow_run' && format('nightly {0}', github.event.workflow_run.created_at) || github.event_name }})
+
+on:
+  workflow_run:
+    workflows: [Nightly CI build]
+    types: [completed]
+    branches: [main]
+  workflow_dispatch:
+    inputs:
+      BASE_IMAGE:
+        type: string
+        description: 'Upstream Jax mealkit image without $arch-mealkit suffix, e.g., (ghcr.io/nvidia/jax-toolbox-internal:6857094059-upstream-jax). Leaving empty implies ghcr.io/nvidia/jax:mealkit'
```

Comment:
IMHO, that makes sense in case the BASE_IMAGE_AMD64 and BASE_IMAGE_ARM64 is completely different tags. In our case, we always add `-amd64` or `-arm64` to the base image tag (this is and _will be_ our naming convention anyways, right?)